app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+from os import makedirs, path, remove
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command.split()) != 3 or command_list[0] != "mv":
+        raise Exception("Wrong command!")
+    mv, incoming_file, new_file = command_list
+    if "/" in new_file:
+        makedirs(path.dirname(new_file), exist_ok=True)
```

Comment:
Better to do like this, because it will not work in some os (check for "/"
```suggestion
    dir_path, file_name = path.split(new_file)
    if dir_path:
        makedirs(dir_path, exist_ok=True)
```